### ISSUE TITLE
focus should be in "last selected item" when opening with keyboard nav

### ISSUE THREAD

**Issue by kikonen** [kikonen/select_svelte#14]
In native select widget, focus is in "last selected" value when opening selector. Makes easier to navigate long selection via keyboard.

*Cases*
- ARROW_UP
- ARROW_DOWN
- ENTER
- SPACE

**Comment by kikonen** [comment]
#35 partially works around this

**Comment by kikonen** [comment]
Should work better now. Should more-or-less work also in remote fetch cases (assuming latency is not too high)

**Comment by kikonen** [comment]
need to handle also
- MOUSE_CLICK

**Comment by kikonen** [comment]
handle also mouseclick
